### ISSUE TITLE
Can't build for web

### ISSUE THREAD

**Issue by chenglou** [Rigellute/L-system-reasonml#1]
```
 We've found a bug for you!
  /Users/chenglou/Desktop/L-system-reasonml/src/index.re 78:20-28
  
  76 ┆ Draw.stroke(Utils.color(~r=113, ~g=247, ~b=159, ~a=190), env);
  77 ┆ 
  78 ┆ let sentence = Str.split(Str.regexp({||}), state.sentence);
  79 ┆ 
  80 ┆ let rec turtleRun = (lst: list(string)) => {
```

Str not available on web.

**Comment by Rigellute** [comment]
Thanks for letting me know @chenglou. I've removed references to `Str` and created my own `split` function 
```js
let rec split = (acc: list(char), str: string) => {
  	let len = String.length(str);
	let stringList = if (len > 0) {
  		split(acc @ [str.[0]], String.sub(str, 1, len - 1))
  	} else {
      acc
  };

  stringList
};
```

**Comment by Rigellute** [comment]
Fixed in [f7be2ce](https://github.com/Rigellute/L-system-reasonml/commit/f7be2ceffd7de2a469168ae07d6920dd0763cce6)
